src/quemb/molbe/autofrag.py

```diff
@@ -290,13 +306,14 @@ def remove_nonnunique_frags(self, natm: int) -> None:
 
     def to_FragPart(self, mol: Mole, n_BE: int, frozen_core: bool) -> FragPart:
         MISSING = []  # type: ignore[var-annotated]
+        MISSING_PER_FRAG = [[] for _ in range(len(self.AO_per_frag))]  # type: ignore[var-annotated]
```

Comment:
@ShaunWeatherly 
Previously, when there were no edges, `relAO_per_edge` was just a flat empty list rather than a list of empty lists per fragment. This led to a lot of special casing in downstream code.

I changed the output from an empty list to an empty list of empty lists also for `graphgen`.